src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationServiceTest.kt

```diff
@@ -53,9 +55,14 @@ class DraftAdjudicationServiceTest {
         draftAdjudicationRepository,
         reportedAdjudicationRepository,
         prisonApiGateway,
+        offenceCodeLookupService,
         dateCalculationService,
         authenticationFacade
       )
+    whenever(offenceCodeLookupService.getParagraphNumber(2)).thenReturn(OFFENCE_CODE_2_PARAGRAPH_NUMBER)
```

Comment:
The advantage of using a service for the lookup is we can write tests that won't break when the offence code mapping changes

Reply:
Yeah - I was thinking of doing something similar - passing the decisions in instead of looking them up - that way we can pass in a test set and know that its working - not done it yet though